angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-toggle.html

```diff
@@ -5,6 +5,7 @@
       tooltip-trigger="mouseenter"
       tooltip-placement="top"
       tooltip-popup-delay="500"
+      onClick="_gaq.push(['_trackEvent', 'Widgets', 'List Mode', 'List Mode']);"
```

Comment:
Backlogged a more holistic refactor of Google Analytics events usage : https://jira.doit.wisc.edu/jira/browse/MUMUP-1839 .